docs/src/understand/thin-edge-json.md

```diff
@@ -1,52 +1,107 @@
 ---
 title: Thin Edge JSON
 tags: [Concept, MQTT]
-sidebar_position: 1
+sidebar_position: 4
 ---
 
 # Thin Edge JSON format
 
-Thin Edge JSON is a lightweight format used in `thin-edge.io` to represent measurements data.
-This format can be used to represent single-valued measurements, multi-valued measurements
+Thin Edge JSON is a lightweight format used in `thin-edge.io` to represent telemetry data: measurements, events and alarms.
+These data are exchanged over the [MQTT bus](mqtt-bus.md) and sent by the devices and services of a agent.
+
+## Sending Telemetry Data
+
+The `thin-edge.io` framework exposes some MQTT endpoints that can be used by local processes
+to exchange data between themselves as well as to get some data forwarded to the cloud.
+It will essentially act like an MQTT broker against which you can write your application logic.
+Other thin-edge processes can use this broker as an inter-process communication mechanism by publishing and
+subscribing to various MQTT topics.
+Any data can be forwarded to the connected cloud-provider as well, by publishing the data to some standard topics.
+
+### Sending Telemetry Data to thin-edge.io
+
+All topics with the prefix `tedge/` are reserved by `thin-edge.io` for this purpose.
+To send measurements to `thin-edge.io`, the measurements represented in Thin Edge JSON format can be published
+to the `tedge/measurements` topic.
+Other processes running on the thin-edge device can subscribe to this topic to process these measurements.
+
+If the messages published to this `tedge/measurements` topic is not a well-formed Thin Edge JSON,
+then that message won’t be processed by `thin-edge.io`, not even partially,
```

Comment:
```suggestion
then these messages won’t be processed by `thin-edge.io`, not even partially,
```
To align with the plurality in the first sentence.

Reply:
Fixed